### PR TITLE
Enable gallery on content pages

### PR DESCRIPTION
**Pull request by Extravert-ir**
1. Hide images on news list for mobile devices
2. Make all content images responsive ([ONLINE-881](https://jira.reactos.org/browse/ONLINE-881))
3. Add photoswipe to content pages so `{{figure}}` shortcode can be used there as well
4. Simplify/unify styles for `<figure>` and `<figcaption>`

Please, do not merge this PR until discussion ends - I'll convert some more pages to use `{{figure}}` shortcode

### CODE REVIEW

**Comment by Extravert-ir** [comment]
https://pr51.web-content.reactos.org/project-news/reactos-0411-released/ - example of "figured" images on a content page
https://pr51.web-content.reactos.org/gallery/ - new gallery page

File: static/css/hugo-easy-gallery.css [hunk 1]
```diff
@@ -1,144 +1,20 @@
-/*
-Put this file in /static/css/hugo-easy-gallery.css
-Documentation and licence at https://github.com/liwenyip/hugo-easy-gallery/
```
Comment (learn-more) [inline static/css/hugo-easy-gallery.css]: You removed license note?

File: themes/reactos/layouts/partials/shared-list.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 						{{ range $.paginator.Pages }}
 						<section class="post">
 							<div class="row">
-								<div class="col-md-3">
+								<div class="col-sm-3 hidden-xs">
```
Comment (learn-more) [inline themes/reactos/layouts/partials/shared-list.html]: This does not affect the images in the news list on the home page, is that intentional?

Comment (Extravert-ir) [reply themes/reactos/layouts/partials/shared-list.html]: Nope, I forgot that

File: static/css/hugo-easy-gallery.css (hunk 1 above)
Comment (Extravert-ir) [reply static/css/hugo-easy-gallery.css]: I left 2 lines from original css file out of 150, do you think it's needed? :)

**Pull request closed without merging**